packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.slow-spec.ts

```diff
@@ -255,6 +255,19 @@ function compareSearchResults(expected: SearchInWorkspaceExpectation[], actual:
 describe('ripgrep-search-in-workspace-server', function (): void {
     this.timeout(10000);
 
+    // Ignoring search with query provided.
```

Comment:
The following comment should be removed, it does not provide additional information.